ui/analyse/src/study/playerBars.ts

```diff
@@ -13,8 +13,8 @@ interface PlayerNames {
 export default function (ctrl: AnalyseCtrl): VNode[] | undefined {
   const study = ctrl.study;
   if (!study || ctrl.embed) {
```

Comment:
these conditions look weird to me, can't you just move this if (ctrl.embed) return; outside of if (!study || ctrl.embed) { ?

Reply:
yeah that would make more sense, i'll change that now.